synapse/lib/stormtypes.py

```diff
@@ -5364,6 +5368,7 @@ def getObjLocals(self):
     async def _derefGet(self, name):
         return self.valu.get(name)
 
+    @stormfunc(readonly=True)
```

Comment:
If we allow this in a `readonly` runtime it'll just fail elsewhere later in the call stack. I think we can remove this annotation.